app/kumactl/cmd/root.go

```diff
@@ -57,16 +51,29 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 			// once command line flags have been parsed,
 			// avoid printing usage instructions
 			cmd.SilenceUsage = true
-			if kumaBuildVersion != nil && kumaBuildVersion.Version != kuma_version.Build.Version {
-				cmd.Println("Warning: Your kumactl version is " + kuma_version.Build.Version + " which is not the same as " + kumaBuildVersion.Version + " for CP. Update your kumactl.")
-			}
 			if root.IsFirstTimeUsage() {
 				root.Runtime.Config = kumactl_config.DefaultConfiguration()
 				if err := root.SaveConfig(); err != nil {
 					return err
 				}
 			}
-			return root.LoadConfig()
+
+			err = root.LoadConfig()
+			if err != nil {
```

Comment:
nit: you can shorten this expression with `if err := root.LoadConfig(); err != nil {`

Reply:
Will do.